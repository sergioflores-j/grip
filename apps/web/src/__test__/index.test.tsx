import { ReactElement } from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import Index from '../pages';

export const shallowRender = (component: ReactElement) => {
  const renderer = createRenderer();
  renderer.render(component);

  const result = renderer.getRenderOutput();

  return { result };
};

describe('Index', () => {
  it('should render correctly', () => {
    const { result } = shallowRender(<Index />);
    expect(result).toMatchSnapshot();
  });
});
