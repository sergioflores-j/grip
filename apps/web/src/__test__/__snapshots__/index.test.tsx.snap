// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render correctly 1`] = `
<div>
  <h1>
    Web
  </h1>
  <Button />
</div>
`;
